packages/docs/package.json

```diff
@@ -16,7 +16,7 @@
     "build.showcase": "pnpm node scripts/showcase.js",
     "start": "pnpm dev",
     "codesandbox.sync": "tsm codesandbox.sync.ts",
-    "deploy": "wrangler pages publish ./dist --branch preview"
+    "deploy": "wrangler pages publish ./dist --branch production"
```

Comment:
Ugh! The actual fix is to remove option this altogether. That will result in the project using the current git branch. Then by default if you are on the main branch (and main has been selected as the production branch) this will deploy a production site. Otherwise, on any other branch it will deploy a preview site.

See https://developers.cloudflare.com/pages/platform/branch-build-controls/#production-branch-control